src/elements/Header/Header.js

```diff
@@ -22,9 +23,10 @@ export default class Header extends Component {
   static H5 = HeaderH5;
   static H6 = HeaderH6;
   static Subheader = HeaderSubheader;
+
   render() {
     return (
-      <_Header {...this.props} _headerElement='div'/>
+      <_Header {...this.props} />
```

Comment:
Seeing as this is done internally and you know all of the props being passed to it, I wonder if it would be (marginally) faster (meaning performance) to just apply them yourself rather than using the spread operator.